### PR TITLE
ci: Update devbox install action org name

### PR DESCRIPTION
**Pull request by mikolajb**
**What problem does this PR solve?**:
The org of an action we use changed, let's update it so we can continue using the action.

**Which issue(s) does this PR fix?**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] If the PR adds a version bump, ensure there is no breaking change in Licensing model (or NA).
- [ ] If a chart is changed or app configuration is significantly changed, the chart version is correctly incremented (so that apps are not automatically upgraded from a previous version of DKP).

### CODE REVIEW

File: .github/workflows/create-dev-tag.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (jimmidyson) [inline .github/workflows/create-dev-tag.yml:42]: You can upgrade to v0.9.0 as well :)

Comment (gracedo) [inline .github/workflows/create-dev-tag.yml:42]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/flux-update-scheduled-check.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/flux-update-scheduled-check.yml:24]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/lint.yaml [hunk 3]
```diff
@@ -24,7 +24,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/lint.yaml:27]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/manifest-validation.yml [hunk 4]
```diff
@@ -26,7 +26,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/manifest-validation.yml:29]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -27,7 +27,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/release.yml:30]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/test.yaml [hunk 6]
```diff
@@ -19,7 +19,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/test.yaml:22]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/update-service-labeler.yaml [hunk 7]
```diff
@@ -31,7 +31,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/update-service-labeler.yaml:34]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/validate-licenses.yaml [hunk 8]
```diff
@@ -28,7 +28,7 @@ jobs:
         uses: cachix/install-nix-action@v26
 
       - name: Install devbox
-        uses: jetpack-io/devbox-install-action@v0.8.0
+        uses: jetify-com/devbox-install-action@v0.8.0
```
Comment (gracedo) [inline .github/workflows/validate-licenses.yaml:31]:
```suggestion
        uses: jetify-com/devbox-install-action@v0.9.0
```

File: .github/workflows/create-dev-tag.yml (hunk 1 above)
Comment (gracedo) [reply .github/workflows/create-dev-tag.yml]: https://github.com/mesosphere/kommander-applications/pull/2100/commits/b8c930ac5feb6068de25c119a978db7e2ddbfe89

**Review by gracedo — APPROVED** [review]

**Pull request merged**